core/templates/dev/head/pages/exploration_editor/EditorServices.js

```diff
@@ -1292,308 +1292,6 @@ oppia.constant('STATE_ERROR_MESSAGES', {
-            if (stateWarnings.hasOwnProperty(unreachableStateName)) {
-              stateWarnings[unreachableStateName].push(
-                STATE_ERROR_MESSAGES.STATE_UNREACHABLE);
-            } else {
-              stateWarnings[unreachableStateName] =
-                [STATE_ERROR_MESSAGES.STATE_UNREACHABLE];
-            }
-          });
-        } else {
-          // Only perform this check if all states are reachable.
-          var deadEndStates = _getUnreachableNodeNames(
-            _graphData.finalStateIds, _graphData.nodes,
-            _getReversedLinks(_graphData.links), false);
-          if (deadEndStates.length) {
-            angular.forEach(deadEndStates, function(deadEndState) {
-              if (stateWarnings.hasOwnProperty(deadEndState)) {
-                stateWarnings[deadEndState].push(
-                  STATE_ERROR_MESSAGES.UNABLE_TO_END_EXPLORATION);
-              } else {
-                stateWarnings[deadEndState] = [
-                  STATE_ERROR_MESSAGES.UNABLE_TO_END_EXPLORATION];
-              }
-            });
-          }
-        }
-
-        _warningsList = _warningsList.concat(_verifyParameters([
-          _graphData.initStateId]));
-      }
-
-      if (Object.keys(stateWarnings).length) {
-        var errorString = (
-          Object.keys(stateWarnings).length > 1 ? 'cards have' : 'card has');
-        _warningsList.push({
-          type: WARNING_TYPES.ERROR,
-          message: (
-            'The following ' + errorString + ' errors: ' +
-            Object.keys(stateWarnings).join(', ') + '.')
-        });
-      }
-
-      var statesWithAnswerGroupsWithEmptyClassifiers = (
-        _getStatesAndAnswerGroupsWithEmptyClassifiers());
-      statesWithAnswerGroupsWithEmptyClassifiers.forEach(function(result) {
-        var warningMessage = 'In \'' + result.stateName + '\'';
-        if (result.groupIndexes.length !== 1) {
-          warningMessage += ', the following answer groups have classifiers ';
-          warningMessage += 'with no training data: ';
-        } else {
-          warningMessage += ', the following answer group has a classifier ';
-          warningMessage += 'with no training data: ';
-        }
-        warningMessage += result.groupIndexes.join(', ');
-
-        _warningsList.push({
-          message: warningMessage,
-          type: WARNING_TYPES.ERROR
-        });
-      });
-    };
-
-    return {
-      countWarnings: function() {
-        return _warningsList.length;
-      },
-      getAllStateRelatedWarnings: function() {
-        return stateWarnings;
-      },
-      getWarnings: function() {
-        return _warningsList;
-      },
-      hasCriticalWarnings: function() {
-        return hasCriticalStateWarning || _warningsList.some(function(warning) {
-          return warning.type === WARNING_TYPES.CRITICAL;
-        });
-      },
-      updateWarnings: function() {
-        _updateWarningsList();
-      }
-    };
-  }
-]);
 
```

Comment:
Remove this line(1295). (Reason: 1294 is also a new line)
I would request you to do the same for line 1156 & 1157.